containerz/containerz.proto

```diff
@@ -417,6 +417,30 @@ message StartContainerRequest {
   RunAs run_as = 11;
 
   map<string, string> labels = 12;
+  
+  // Limits defines restrictions or limits upon this container.
+  message Limits {
+    // max_cpu specifies the maximum amount of cpu this container can use.
+    // For example a value of 0.5 means this container can use a maximum of
+    // half a cpu. This setting does not imply that the container is pinned
+    // to a cpu but rather only the maximum amount of cpu it can use.
+    double max_cpu = 1;
+
+    // soft_mem defines a soft memory limit. If the system is not memory
+    // constrained this limit has no effect. If it is memory constrained
+    // this container may be oom killed if it exceed this limit. This value
+    // is effectively a soft memory limit. This limit should be smaller than
+    //  max_mem otherwise it will have no effect.
+    int64 soft_mem =2;
```

Comment:
What are the units here? Generally, I think best practice is to have the units in the name -- e.g., `soft_mem_bytes`.

Reply:
ah yes - thanks!